dev/TreeView/TreeViewItem.xaml

```diff
@@ -120,6 +141,7 @@
                                 <CheckBox x:Name="MultiSelectCheckBox"
                                             Width="32"
                                             MinWidth="32"
+                                            MinHeight="{ThemeResource TreeViewItemMinHeight}"
```

Comment:
You should name this TreeViewItemMultiSelectCheckBoxMinHeight now that this is only for the checkbox.